extensions/wikia/Email/EmailController.class.php

```diff
@@ -175,11 +187,21 @@ protected function getReplyToAddress() {
 	 * @return string
 	 */
 	protected function getBody() {
-		$html = $this->app->renderView(
+		$moduleBody = $this->app->renderView(
 			get_class( $this ),
 			'body',
 			$this->request->getParams()
 		);
+
+		// todo: stop grunt-email-builder from adding these extraneous tags in to the partials
```

Comment:
Opened this issue https://github.com/Automattic/juice/issues/110 to see if it can be addressed at the Juice level.